lib/tmuxinator/project.rb

```diff
@@ -31,7 +31,7 @@ def root
 
     def name
       name = yaml["project_name"] || yaml["name"]
-      name.shellescape
+      name.blank? ? nil : name.shellescape
```

Comment:
Went with blank because that's what the #root fn did, but I don't think it
makes a difference.
On Wed, May 13, 2015 at 3:43 AM maxmeyer <notifications@github.com> wrote:

> In lib/tmuxinator/project.rb
> <https://github.com/tmuxinator/tmuxinator/pull/303#discussion_r30208295>:
>
> > @@ -31,7 +31,7 @@ def root
> >
> >      def name
> >        name = yaml["project_name"] || yaml["name"]
> > -      name.shellescape
> > +      name.blank? ? nil : name.shellescape
>
> what about name.empty?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tmuxinator/tmuxinator/pull/303/files#r30208295>.
>